.changeset/late-garlics-sneeze.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
"we now build with TS5, we don't think it affects callers though" might be reasonable just in case it does break things!